android/src/main/java/com/clevertap/clevertap_plugin/CleverTapAppContextHolder.java

```diff
@@ -0,0 +1,19 @@
+package com.clevertap.clevertap_plugin;
+
+
+import android.content.Context;
+import android.util.Log;
+
+public class CleverTapAppContextHolder {
```

Comment:
@shivamsharma2710 please add javadoc on what this class does

Reply:
Done.